src/languages/es.js

```diff
@@ -1099,6 +1099,7 @@ export default {
         visibility: 'Visibilidad',
         restrictedDescription: 'Sólo las personas en tu espacio de trabajo pueden encontrar esta sala',
         privateDescription: 'Sólo las personas que están invitadas a esta sala pueden encontrarla',
+        publicDescription: 'Cualquiera puede encontrarlo',
```

Comment:
@marcochavezf do you mind checking this translation for me :D

Reply:
LGMT 👍🏽 and thanks for fixing the issue that you found!